manifests/init.pp

```diff
@@ -40,10 +40,15 @@
   Optional[String[1]] $service_name = undef,
   Optional[String[1]] $package_source = undef,
   Optional[String[1]] $package_provider = undef,
+  Optional[Boolean] $manage_service_file = undef,
```

Comment:
Yeah, that actually makes a lot more sense... I guess we could make it default to false, non-optional